### DISCUSSION TITLE
Welcome to PDS-Data-Dictionaries.github.io Discussions!

### DISCUSSION THREAD

**Discussion by jordanpadams** [pds-data-dictionaries/PDS-Data-Dictionaries.github.io#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sarabond** [comment]
Hello! Thank you for starting this, Jordan!

**Comment by katecrombie** [comment]
Hello all.

**Comment by acraugh** [comment]
Hello! I'm adding "Add a picture to your GitHub" profile to my Trello board. Am I cool yet? 😎

**Reply by wkiri** [reply]
You will be, when you add the picture! ;)

**Reply by acraugh** [reply]
Coolness achieved!

**Reply by wkiri** [reply]
Wow, that is an awesome picture :)
